backend/src/test/java/org/cryptomator/hub/api/VaultResourceIT.java

```diff
@@ -286,6 +296,7 @@ public void testCreateVault3() {
 		@Test
 		@Order(2)
 		@DisplayName("PUT /vaults/7E57C0DE-0000-4000-8000-000100003333 returns 200, updating only name, description and archive flag")
+		@DBRollback
```

Comment:
No, because this is a different nested test.

I added an additional db rollback extenstion in https://github.com/cryptomator/hub/pull/266/commits/4f023e0ce15588f06fead5512294c41af2cccf77, to indicate wether the db should be reset before or after the test.